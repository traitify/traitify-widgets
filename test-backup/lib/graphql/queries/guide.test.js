import guideQuery from "lib/graphql/queries/guide";

describe("Graphql", () => {
  describe("Queries", () => {
    describe("Guide", () => {
      describe("get", () => {
        it("creates query", () => {
          const query = guideQuery({
            params: {assessmentId: "xyz", localeKey: "es-US"},
            fields: ["name", {competencies: ["id", "name"]}]
          });

          expect(query).toMatchSnapshot();
        });

        it("creates query with defaults", () => {
          const query = guideQuery({params: {assessmentId: "xyz", localeKey: "es-US"}});

          expect(query).toMatchSnapshot();
        });
      });
    });
  });
});
