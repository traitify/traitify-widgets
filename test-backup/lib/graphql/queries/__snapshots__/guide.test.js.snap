// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graphql Queries Guide get creates query 1`] = `"{ guide(assessmentId:"xyz",localeKey:"es-US") { name competencies { id name } }}"`;

exports[`Graphql Queries Guide get creates query with defaults 1`] = `"{ guide(assessmentId:"xyz",localeKey:"es-US") { competencies { id name introduction order questionSequences { id name personalityTypeId questions { id text adaptability order purpose } } } }}"`;
